app/Dockerfile

```diff
@@ -56,9 +56,14 @@ RUN npm run build -- --no-lint
 # Run the Next.js server
 # =====================================
 # Use clean image for release, excluding any unnecessary files or dependencies
-FROM node:18-alpine AS release
+FROM node:18-bullseye-slim AS release
 WORKDIR /app
 
+# As of July 13, 2023, 18-bullseye-slim has a vulnerability in the version of
+# semver that it uses (GHSA-c2qf-rxjj-qqgw). Update npm to avoid the vulnerability
+# being caught in vulnerability scans.
+RUN npm install -g npm@latest
```

Comment:
Looks like removing `npm` worked, tested it out here: https://github.com/navapbc/platform-test-nextjs/actions/runs/5545995357/jobs/10125691478

This stage doesn't technically need `npm` so I think we can remove it to fix this and avoid any future vulnerabilities in it:

```suggestion
# Release stage doesn't have a need for `npm`, so remove it to avoid 
# any vulnerabilities specific to NPM
RUN npm uninstall -g npm
```